package.json

```diff
@@ -941,7 +941,8 @@
     "jsonwebtoken": "^9.0.0",
     "jsts": "^1.6.2",
     "kea": "^2.6.0",
-    "langchain": "^0.0.151",
+    "langchain": "^0.0.186",
+    "langsmith": "^0.0.48",
```

Comment:
Consider making `langsmith` a dev dependency if that's appropriate

Reply:
So turns out it's actually a [langchain dependency](https://github.com/langchain-ai/langchainjs/blob/95f8563d4388f72074fa97a063c1b792b0bbfc0f/langchain/package.json#L1372), so not going to get around this one. I added it as an explicit dependency since we were importing things from it directly (langsmith client for fetching test data and writing runs).